CHANGELOG.next.asciidoc

```diff
@@ -518,6 +518,28 @@ field. You can revert this change by configuring tags for the module and omittin
 - Improve ECS categorization field mappings in azure module. {issue}16155[16155] {pull}19376[19376]
 - Add automatic retries and exponential backoff to httpjson input. {pull}18956[18956]
 - Add text & flattened versions of fields with unknown subfields in aws cloudtrail fileset. {issue}18866[18866] {pull}19121[19121]
+- Added Microsoft Defender ATP Module. {issue}17997[17997] {pull}19197[19197]
+- Add experimental dataset tomcat/log for Apache TomCat logs {pull}19713[19713]
```

Comment:
Looks like there was a merge issue that needs corrected here.

Reply:
We fixed these during merging, together with @adriansr. I think as long as the RSA merges first with backport it should fix itself?